worker/src/DepUsrSCTP.cpp

```diff
@@ -40,7 +41,7 @@ inline static void sctpDebug(const char* format, ...)
 	va_list ap;
 
 	va_start(ap, format);
-	vsprintf(buffer, format, ap);
+	vsnprintf(buffer, 10000, format, ap);
```

Comment:
10000 characters? what's the rationale for this change?

Reply:
No change here other than `vsnprintf` instead of `vsprintf`. Buffer was already 10000 chars for whatever reason:
![CleanShot-2022-11-30-at-18 36 39@2x](https://user-images.githubusercontent.com/16191/204868776-eb27af9c-64b7-413b-a4b6-23c1b1695f99.png)

Reply:
The meme should say:

"While Making a PR I make unrelated changes just because, and reviewers must expand the diffs to understand my rationale and :0 "

Reply:
🤣🤣🤣🤣🤣

Reply:
BTW, that change is error prone. It should be `vsnprintf(buffer, sizeof(buffer), format, ap)`

Reply:
Done: https://github.com/versatica/mediasoup/pull/960/commits/40195d5a2c23c85d3339948d5fa8b934056f6d65